warrant/aws_srp.py

```diff
@@ -90,6 +91,17 @@ def calculate_u(big_a, big_b):
     return hex_to_long(u_hex_hash)
 
 
+def current_time():
+    """Return the current time formatted per Cognito rules."""
+    if sys.platform.startswith('win'):
+        return datetime.datetime.utcnow().strftime("%a %b %#d %H:%M:%S UTC %Y")
+    elif platform.libc_ver()[0] == 'glibc':
+        return datetime.datetime.utcnow().strftime("%a %b %-d %H:%M:%S UTC %Y")
+    else:
+        timestamp = datetime.datetime.utcnow().strftime("%a %b %d %H:%M:%S UTC %Y")
```

Comment:
It seems that those 2 lines are enough to work on Windows, Linux and non-glibc systems. I think we can delete check for platforms and leave only those 2 lines.